### PR TITLE
Admin UI: Initialize flutter setup

### PR DESCRIPTION
**Pull request by jkoenig134**
# Readiness checklist

-   [ ] I added/updated unit tests.
-   [ ] I added/updated integration tests.
-   [x] I ensured that the PR title is good enough for the changelog.
-   [x] I labeled the PR.

### CODE REVIEW

**Comment by tnotheis** [comment]
I close it until it is fully implemented, to keep the PR overview clean.

**Comment by tnotheis** [comment]
Yeah, I thought about that. But I would at least like to have the pipe setup completed, and I won't manage to do this today and maybe not even tomorrow

**Comment by tnotheis** [comment]
I didn't say anything about WHICH pipeline. ;) 
But at least the build and test pipelines should exist.

**Comment by jkoenig134** [comment]
ah, right.. good catch. should I add these?

**Comment by tnotheis** [comment]
Maybe you can give me the commands. Then I will add the pipelines once I find time.

**Comment by jkoenig134** [comment]
```
name: Test

on:
  push:

jobs:
  run-checks:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v3
      - uses: subosito/flutter-action@v2.8.0
      - run: flutter pub get
      - run: flutter analyze
      - run: dart format . --set-exit-if-changed -l 150
```

File: AdminUi/.gitignore [hunk 1]
```diff
@@ -0,0 +1,9 @@
+pubspec_overrides.yaml
+.dart_tool
+build
+.flutter-plugins
+.flutter-plugins-dependencies
+.DS_Store
+
+# config.json for connecting to the admin api
+config.json
```
Comment (tnotheis) [inline AdminUi/.gitignore:9]: @jkoenig134 Shouldn't we commit this with the connection to the local server?

**Review by tnotheis — APPROVED** [review]
@jkoenig134 I skimmed the new files and I think I'm okay with it. But you should give your final Go before we merge. (and maybe we should make a real PR out of this draft 😉)

File: AdminUi/.gitignore (hunk 1 above)
Comment (tnotheis) [reply AdminUi/.gitignore:9]: With "this" I mean the `config.json` file

Comment (tnotheis) [reply AdminUi/.gitignore:9]: Sounds overcomplicated. Why not just committing the config.json file?
But if that's the way to go in flutter, I'm okay with it.

Comment (jkoenig134) [reply AdminUi/.gitignore:9]: As mac users will have a different config I always have to make sure to not commit it.

**Comment by tnotheis** [comment]
We will add the Dockerfile and the corresponding build and publish pipelines in a separate PR.

**Review by tnotheis — APPROVED** [review]

**Pull request merged**